feconf.py

```diff
@@ -500,9 +500,15 @@ def get_empty_ratings():
 # category.
 MINIMUM_SCORE_REQUIRED_TO_REVIEW = 10
 
+# The number of easy question skill difficulty.
```

Comment:
The difficulty value for "easy" questions.

Have similar docstrings for the other two.

Reply:
Done.

Reply:
Please use the version I provided; yours is grammatically incorrect. (Alternatively, in your version, change "easy" to "an easy".)

Reply:
Sorry for the typo, corrected it.